test/cpp/inference/api/analysis_cpu_quantize_pass_tester.cc

```diff
@@ -0,0 +1,171 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <fstream>
+#include <iostream>
+
+#include "paddle/fluid/inference/api/paddle_analysis_config.h"
+#include "test/cpp/inference/api/tester_helper.h"
+
+#include "paddle/fluid/framework/ir/pass.h"
+#include "paddle/fluid/framework/ir/pass_tester_helper.h"
+#include "paddle/fluid/platform/enforce.h"
+#include "paddle/fluid/framework/ir/graph.h"
+#include "paddle/fluid/framework/op_desc.h"
+#include "paddle/fluid/framework/program_desc.h"
+#include <unordered_map>
+#include <glog/logging.h>
+
+using namespace std;
+
+DEFINE_bool(enable_mkldnn, true, "Enable MKLDNN");
+
+namespace paddle {
+namespace inference {
+namespace analysis {
+
+static float const SCALE = 2.f;
+const std::vector<std::string> PreGraphPasses({
+   "simplify_with_basic_ops_pass",
```

Comment:
maybe only general and conv related patterns are needed.

Reply:
not sure 'general' including, is it ok now?

Reply:
yes:)